backend/src/main/resources/static/aas-registry-openapi.yaml

```diff
@@ -716,11 +716,6 @@ components:
           minLength: 1
           maxLength: 500
     Reference:
-      type: object
-      oneOf:
-      - $ref: '#/components/schemas/GlobalReference'
-      #- $ref: '#/components/schemas/ModelReference'
```

Comment:
What's the comment for? :)

Reply:
Reference was an interface and only GlobalReference inherited it. 
Generate the api class with the new versions failed because of the interface definition. So i decided to use reference as class instead of interface. 
Comment -> I dont know but ModelReference is not used anywhere :)